docs/making_queries.md

```diff
@@ -53,15 +53,15 @@ notes = await Note.objects.filter(completed=True).all()
 
 There are some special operators defined automatically on every column:
 
-* `in` - SQL `IN` operator.
-* `exact` - filter instances matching exact value.
-* `iexact` - same as `exact` but case-insensitive.
-* `contains` - filter instances containing value.
-* `icontains` - same as `contains` but case-insensitive.
-* `lt` - filter instances having value `Less Than`.
-* `lte` - filter instances having value `Less Than Equal`.
-* `gt` - filter instances having value `Greater Than`.
-* `gte` - filter instances having value `Greater Than Equal`.
+- `in` - SQL `IN` operator.
+- `exact` - filter instances matching exact value.
+- `iexact` - same as `exact` but case-insensitive.
```

Comment:
Any reason for this change? I think they're both valid.

Reply:
Idk it's changed after running scripts/lint.

Reply:
Weird, it's not happening for me. Can you check your env and requirements?

Reply:
Oh, It caused by my auto-formatting pylint. I will fix it now.

Reply:
@aminalaee Any updates ?